doc/smart-switch/pmon/smartswitch-pmon.md

```diff
@@ -537,35 +577,9 @@ get_health_info(self):
     Retrieves the dpu health object having the detailed dpu health Fetched from the DPUs
 
     Returns:
-        An object instance of the dpu health. 
+        An object instance of the dpu health as shown in the schema
```

Comment:
As we agreed, get_health_info is not needed. The implementation for querying DPU health information should be in the common code.

Reply:
Agree, this section has been updated.